docs/source/Datasets.rst

```diff
@@ -0,0 +1,147 @@
+Datasets
+========
+.. currentmodule:: hub
+
+Creating Datasets
+~~~~~~~~~~~~~~~~~
+.. autosummary::
+    :toctree:
+    :nosignatures:
+
+    hub.dataset
+    hub.empty
+    hub.like
+    hub.ingest
+    hub.ingest_kaggle
+    hub.ingest_dataframe
+    hub.ingest_huggingface
+
+Loading Datasets
+~~~~~~~~~~~~~~~~
+.. autosummary::
+    :toctree:
+    :nosignatures:
+
+    hub.load
```

Comment:
maybe hub.dataset should be here as well?

Reply:
should it be present both in "creating" as well as "loading"? Left it this way because I think we want people to use hub.load for loading and not dataset